src/ifElse.js

```diff
@@ -0,0 +1 @@
+export default (pred, ifDo, elseDo) => (...args) => pred(...args) ? ifDo(...args) : elseDo(...args);
```

Comment:
Would it make sense to receive an object instead of 3 params or would currying help?

how about `(pred, ifDo, elseDo=noop)` ?